src/main/java/com/puppycrawl/tools/checkstyle/checks/LineSeparatorOption.java

```diff
@@ -42,7 +42,7 @@
      * Matches CR, LF and CRLF line separators.
      * Only the length is used - the actual value is ignored.
      */
-    LF_CR_CRLF("##"),
```

Comment:
comment above should be fixed.

Reply:
Fixed in what regard?
The comment is still valid.
Only the length of the enum argument is used, but the value `#` is ignored.

Reply:
you are right.